docs/api.rst

```diff
@@ -12783,6 +12783,9 @@ Managing custom accounting rules
   :reqjsonarr optional[array[string]] event_types: List of possible event types to use while filtering.
   :reqjsonarr optional[array[string]] event_subtypes: List of possible event subtypes to use while filtering.
   :reqjsonarr optional[array[string]] counterparties: List of possible counterparties to use while filtering. Instead of a string a null value can also be given to mean counterparty being None.
+  :reqjsonarr optional[array[int]] identifiers: List of rule identifiers to filter by. Only rules with these identifiers will be returned.
+  :reqjsonarr optional[bool] only_with_event_ids: If true, only returns rules that have specific event IDs associated with them. Default is false.
```

Comment:
Why provide both only_with_event_ids and event_ids arguments? Wouldn't providing an event_ids array be enough?

Reply:
luki asked for it. he wants to be able to tell apart rules that are explicitly tied to specific events from rules that apply generally, without having to send an `event_ids` array.

Reply:
So as discussed in the standup, I kind of understand what he needed this for. But then the naming would better be something like. "only_custom_rules" ? And then explain what that means in the docstring?

Also make it so that if this is given then if some other arguments don't make sense don't allow them to be given.